packages/taco/src/taco.ts

```diff
@@ -143,6 +143,25 @@ export const decrypt = async (
   porterUri?: string,
   signer?: ethers.Signer,
   customParameters?: Record<string, CustomContextParam>,
+): Promise<Uint8Array> => {
+  const authProviders = makeAuthProviders(provider, signer);
+  return decryptWithAuthProviders(
+    provider,
+    domain,
+    messageKit,
+    authProviders,
+    porterUri,
+    customParameters,
+  );
+};
+
+export const decryptWithAuthProviders = async (
```

Comment:
If we are accepting that we are making backwards incompatible changes for the next release (major version bump), do we need a new method, as opposed to just changing the old method?

Reply:
It wasn't in the cards when I made this PR. I will address necessary changes in the follow-up PR or close this PR without merging.